front/lib/actions/mcp_actions.ts

```diff
@@ -837,7 +838,10 @@ export async function listToolsForServerSideMCPServer(
     // Create configurations and add required properties.
     const serverSideToolConfigs = makeServerSideMCPToolConfigurations(
       config,
-      rawTools
+      rawTools.map((tool) => ({
+        ...tool,
+        retryPolicy: DEFAULT_MCP_TOOL_RETRY_POLICY,
```

Comment:
Maybe more clear if we set the default in the previous loop, as for stakeLevel and availability ?